ashlar/ashlar_server/settings.py

```diff
@@ -110,5 +114,34 @@
 
 STATIC_URL = '/static/'
 
+# Allow CORS requests from any port on the loopback interface
+CORS_ORIGIN_REGEX_WHITELIST = (
+    r'^http://127.0.0.1:\d+',
```

Comment:
As we discovered, this needs to allow (at the least) `localhost` as well, but it may be worth finding a way to disable it completely on develop (perhaps keying off `DEBUG`?).